### ISSUE TITLE
feat(CdkSort): Creation of a module

### ISSUE THREAD

**Issue by dev054** [angular/components#20157]
#### Feature Description
A `CdkSort` module as an unopinionated version of MatSort.

#### Use Case
Possibility of customize the icons (like requested in https://github.com/angular/components/issues/16560) and others.

PS: [As answered by @jelbourn](https://github.com/angular/components/issues/16560#issuecomment-512932493), you already have plans to do this, but as I couldn't find any issue for this feature, I decided to create it to make sure it’s on you feature track :)

**Comment by jelbourn** [comment]
Yeah, this is in our backlog (which we have in a jira, but can keep this open on here for comments)

**Comment by crisbeto** [comment]
We are closing this issue, because we've determined, using a semi-automated process, that it is no longer relevant, has low engagement, or it does not have enough information to proceed. Feel free to comment if you think the issue should be reopened, or open a new issue with the latest version of our code instead.
